### DISCUSSION TITLE
Google Map Field (Pro)

### DISCUSSION THREAD

**Discussion by manfield** [Laravel-Backpack/community-forum#1166] [Q&A (Help)] [answered]
Hi community,
I'm using the google_map field from Backpack Pro.
Is there a way to set the default zoom level?

**Comment by siberfx** [comment, marked as answer]
yes, publish the field file and neither make it through a variable via controller, nor fixed value on google_map file
![{B685C1E7-59D3-4D8A-8E78-CA3F4C9DE690}](https://github.com/user-attachments/assets/5b3bd774-71c0-40dd-9e8d-5d552a35991e)
as in the image

**Reply by manfield** [reply]
Thanks a lot
